client/core/src/com/cyberbot/bomberman/controllers/NetworkedGameplayController.java

```diff
@@ -0,0 +1,101 @@
+package com.cyberbot.bomberman.controllers;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.World;
+import com.badlogic.gdx.utils.Disposable;
+import com.cyberbot.bomberman.core.controllers.LocalWorldController;
+import com.cyberbot.bomberman.core.controllers.SnapshotQueue;
+import com.cyberbot.bomberman.core.models.Updatable;
+import com.cyberbot.bomberman.core.models.entities.PlayerEntity;
+import com.cyberbot.bomberman.core.models.net.Connection;
+import com.cyberbot.bomberman.core.models.net.data.PlayerData;
+import com.cyberbot.bomberman.core.models.net.packets.PlayerSnapshotPacket;
+import com.cyberbot.bomberman.core.models.tiles.MissingLayersException;
+import com.cyberbot.bomberman.core.models.tiles.TileMap;
+import com.cyberbot.bomberman.models.Drawable;
+import com.cyberbot.bomberman.models.KeyBinds;
+import com.cyberbot.bomberman.net.NetService;
+
+import java.util.InvalidPropertiesFormatException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+import static com.cyberbot.bomberman.core.utils.Constants.*;
+
+public class NetworkedGameplayController implements Updatable, Drawable, Disposable {
+    private final PlayerEntity localPlayer;
+
+    private final TextureController textureController;
+    private final InputController inputController;
+
+    private final LocalWorldController worldController;
+    private final TileMap map;
+
+    private final SnapshotQueue snapshotQueue;
+    private final NetService netService;
+
+    private final ScheduledExecutorService snapshotService;
+    private final ScheduledExecutorService inputPollService;
+
+    public NetworkedGameplayController(PlayerData player, String mapPath, Connection connection)
+        throws MissingLayersException, InvalidPropertiesFormatException {
+        KeyBinds binds = new KeyBinds(); // TODO: Load from preferences
+
+        World world = new World(new Vector2(0, 0), false);
+        map = new TileMap(world, mapPath);
```

Comment:
Legacy constructor is used to create a tile map. Newer Factory method should be used.